src/k8s_client/k8s_client.go

```diff
@@ -204,10 +203,6 @@ func (c *k8sClient) ListMachines() (*machinev1beta1.MachineList, error) {
 }
 
 func (c *k8sClient) PatchEtcd() error {
```

Comment:
This is called by the installer, not the controller as you may have suggested in your Slack messages, so I think you'd have to put that back

Reply:
https://github.com/openshift/assisted-installer/blob/c9c39a06fdcd172a26cd61590cb96bffa031c5b1/src/installer/installer.go#L393

Reply:
But isn't the installer client mocked here https://github.com/openshift/assisted-installer/blob/c9c39a06fdcd172a26cd61590cb96bffa031c5b1/src/main/main.go#L69 ?

So this code is not actually called by the installer

Reply:
Anyhow, it is never called because Etcd patch is never required (only version less than 4.7)

https://github.com/openshift/assisted-installer/blob/c9c39a06fdcd172a26cd61590cb96bffa031c5b1/src/installer/installer.go#L387

Reply:
We can remove it once we don't need to support 4.6 installations.
Currently, we do:
https://console.redhat.com/openshift/assisted-installer/clusters/~new

Reply:
I basically was talking about dry mode flow